File: nearbyfundi_backend/app/Models/PrivacyPolicy.php
<?php

namespace App\Models;

use Illuminate\Database\Eloquent\Model;

class PrivacyPolicy extends Model
{
    protected $table = 'privacy_policies';

    protected $fillable = [
        'content',
    ];

    protected $casts = [
        'created_at' => 'datetime',
        'updated_at' => 'datetime',
    ];
}

File: nearbyfundi/lib/models/privacy_policy.dart
class PrivacyPolicy {
  final int id;
  final String content;
  final DateTime? updatedAt;

  PrivacyPolicy({required this.id, required this.content, this.updatedAt});

  factory PrivacyPolicy.fromJson(Map<String, dynamic> json) => PrivacyPolicy(
    id: json['id'] ?? 0,
    content: json['content'] ?? '',
    updatedAt: json['updated_at'] != null ? DateTime.parse(json['updated_at']) : null,
  );
}